guacamole/src/main/frontend/src/app/import/services/connectionParseService.js

```diff
@@ -167,7 +196,7 @@ angular.module('import').factory('connectionParseService',
 
             // Set the parent identifier now that it's known
             return {
-                ...connection,
+                ..._.omit(connection, 'group'),
```

Comment:
This isn't strictly necessary - when passed to the `Connection` constructor later, this will automatically get removed, and will actually incur a small performance penalty from copying the connection object.

If people don't think the added cleanliness of removing the `group` field at this point is worth it, I'd be ok with removing this.

Reply:
If the performance penalty is negligible and doesn't change depending on the number of connections you import, then I'm fine leaving it. If the performance penalty scales as you increase the number of connections, then it seems like removing this is probably a good idea, particularly if it is going to get taken off elsewhere.